rust/stracciatella/src/schemas/yaml/game.schema.yaml

```diff
@@ -234,6 +241,41 @@ properties:
       Multiplier for damage that is inflicted on the legs
 
       Vanilla setting: `0.5`
+  aim_bonus_per_std_ap:
+    type: number
```

Comment:
I think we should use `$ref: types/int8.schema.yaml` for these, to guard against overflow - if somebody decides they want a really high value for these.

Reply:
That's a design question to me: where do we draw the line? If they want to break the file, the game, they have countless options already.

Reply:
Maybe always use the same numeric value as the backing C++ code?

I was looking at `range_bonus_barrel_extender`. The default is `100`. It is not unreasonable to raise it to say `150`, and that already causes overflow.

Reply:
Good point, it should be bigger on the engine side already.